src/pages/fr/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,698 @@
+
+<UIFrameworkTabs>
+  <Fragment slot="preact">
+  ```shell
+  npm install nanostores @nanostores/preact
+  ```
+  </Fragment>
+  <Fragment slot="react">
+  ```shell
+  npm install nanostores @nanostores/react
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```shell
+  npm install nanostores @nanostores/solid
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+  ```shell
+  npm install nanostores
+  ```
+  :::note
+  Pas de package d'assistance ici ! Nano Stores peuvent être utilisés comme Svelte stores standard
+
+  :::
+  </Fragment>
+  <Fragment slot="vue">
+  ```shell
+  npm install nanostores @nanostores/vue
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+Vous pouvez aller directement dans le [Guide d'utilisation de Nano Stores](https://github.com/nanostores/nanostores#guide) à partir d'ici, ou suivez notre exemple ci-dessous!
+
+## Exemple d'utilisation - Menu déroulant pour un panier de ecommerce
+
+Disons que nous construisons une interface de ecommerce simple avec trois éléments interactifs :
+-Un formulaire de soumission "ajouter au panier"
+-Un menu déroulant du panier pour afficher les articles ajoutés
+-Une bascule de chariot
+
+<LoopingVideo sources={[{ src: '/public/videos/stores-example.mp4', type: 'video/mp4' }]} />
+
+_[**Essayez l'exemple complété**](https://github.com/withastro/astro/tree/main/examples/with-nanostores) sur votre machine ou en ligne via Stackblitz._
+
+Votre fichier Astro de base peut ressembler à ceci:
+
+```astro
+---
+// Exemple: src/pages/index.astro
+import CartFlyoutToggle from '../components/CartFlyoutToggle';
+import CartFlyout from '../components/CartFlyout';
+import AddToCartForm from '../components/AddToCartForm';
+---
+
+<!DOCTYPE html>
+<html lang="en">
+<head>...</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">Astro storefront</a>
+      <CartFlyoutToggle client:load />
+    </nav>
+  </header>
+  <main>
+    <AddToCartForm client:load>
+    <!-- ... -->
+    </AddToCartForm>
+  </main>
+  <CartFlyout client:load />
+</body>
+</html>
+```
+
+### Utilisant "atoms"
+
+Commençons par ouvrir notre `Cart Flyout` chaque fois que `Car Flyout Toggle` est cliqué. 
+
+Tout d'abord, créez un nouveau fichier JS ou TS pour contenir notre store. Nous utiliserons un
+ ["atom"](https://github.com/nanostores/nanostores#atoms) pour cela:
+
+```js
+// src/cartStore.js
+import { atom } from 'nanostores';
+
+export const isCartOpen = atom(false);
+```
+
+Maintenant, nous pouvons importer ce store dans n'importe quel fichier qui doit être lu ou écrit. Nous allons commencer par câbler notre `CartFlyoutToggle`:
+
+<UIFrameworkTabs>
+<Fragment slot="preact">
+```jsx
+// src/components/CartFlyoutToggle.jsx
+import { useStore } from '@nanostores/preact';
+import { isCartOpen } from '../cartStore';
+
+export default function CartButton() {
+  // lire la valeur du magasin avec le crochet `useStore` const $isCartOpen = useStore(isCartOpen);
```

Comment:
As per the french version of the react docs, "hooks" shouldn't be translated
```suggestion
  // lire la valeur du store avec le _hook_ `useStore`
  const $isCartOpen = useStore(isCartOpen);
```

Reply:
```suggestion
  // lire la valeur du store avec le hook `useStore`
  const $isCartOpen = useStore(isCartOpen);
```